src/styles/italic/types.ts

```diff
@@ -0,0 +1,3 @@
+type ItalicStyles = 'normal' | 'italic';
```

Comment:
```suggestion
type ItalicStyles = (TextStyle | CSSProperties)['fontStyle'];
```

In this way we have options for both native + web platforms.